taca/analysis/analysis.py

```diff
@@ -359,7 +377,7 @@ def run_bcl2fastq(run, config):
             cl.extend(['--adapter-stringency', adapter_stringency])
         aggregated_tiles = cl_options.get('aggregated-tiles')
         if aggregated_tiles and aggregated_tiles in ['AUTO', 'YES', 'NO']:
-            cl.etend(['--aggregated-tiles', aggregated_tiles])
+            cl.extend(['--aggregated-tiles', aggregated_tiles])
```

Comment:
Don't thank me - PyLint would never let that slip through!

If your interested, I'm sure Atom can handle such a plugin *just as well* as Sublime, hehe :smirk: